docs/_posts/2020-01-09-new-year-new-bolt.md

```diff
@@ -0,0 +1,28 @@
+---
+title: New Year, New Bolt
+---
+
+Welcome to 2020!
+
+The team has been ~~on vacation~~ hard at work since the last update, adding a bevy of new features as well as formalizing and sanding the rough edges of existing features to prepare for the imminent Bolt 2.0 release.
+
+A bulk of the work has been finalizing the v2 inventory, which will fittingly be the inventory API for Bolt 2.0. You can check out [the previous update](2019-09-26-the-road-to-bolt-2) for some more info on what that means.
+
+We've made a number of enhancements to the plugin system, including adding more out-of-the-box integrations. These now include Azure, AWS, Terraform, and Vault among others.
+
+Plugins can now be configured in both `bolt.yaml` and `inventory.yaml` using _other_ plugins. For instance, you can use the Vault plugin to lookup credentials to connect to AWS.
+
+Several new commands have joined the CLI, including `bolt project init` and `bolt project migrate` to create and manage Bolt projects, as well as `bolt group show` to see the targets in a group.
+
+## Bolt 2.0
+
+The Bolt 2.0 release will be coming soon and we encourage everyone to upgrade. The `bolt project migrate` command will automatically translate your Bolt inventory file to the v2 format, which is a great way to prepare for the release.
+
+The backward-incompatible changes will be minimal and primarily target under-used or downright unusable features. In particular, the changes include:
+
+* The `bolt-inventory-pdb` command will be removed in favor of the `puppetdb` inventory plugin
+* The `--nodes` flag will be replaced with `--targets`
+* v1 inventory files will no longer be supported (use `bolt project migrate` to convert your inventory file)
+* The low-level API for dynamically creating and modifying `Target` objects in a plan has been cleaned up and now works reliably
```

Comment:
It might be worth adding plan language updates here too, particularly since I think those will be hardest to debug once 'future: true' is set:
* Remove `node` from Result status_hash (read: the JSON representation of the Result)
* Rename `value` to `result` in the Result JSON representation

And I think one other:
* Filepaths will expand relative to the Boltdir

I know this isn't an exhaustive list, so it's fine to leave as is if 7 items seems too long. We'll have a canonical list in the docs when we release (I assume? I hope?)

Reply:
I definitely want to have a canonical list including all the changes to the plan API. I don't want to go into too great of detail here, mostly trying to explain the overall _shape_ of the release and why it exists.